yapapi/network.py

```diff
@@ -89,6 +111,7 @@ async def create(
         # add requestor's own address to the network
         await network.add_owner_address(network.owner_ip)
 
+        network._state_machine.create()
```

Comment:
why is it here twice?

Reply:
This follows the `create` lifecycle - the first call advances the state machine from `initialized` to `creating`, the second one from `creating` to `ready`.
The approach taken here is analogous to how the `Service` lifecycle is defined (`ServiceState.lifecycle`).

Reply:
I think it would be clearer to have two separate, explicit transitions here then -> `initialize`? that moves from `initialized` to `creating` and `create` that moves from `creating` to `ready`

Reply:
or maybe `create` -> `creating`, `start` -> `ready` ? ...

Reply:
Updated in b319b73